### ISSUE TITLE
Payment Link to Card Payment on my website home page linked to Viva Wallet

### ISSUE THREAD

**Issue by highvalueasset** [VivaPayments/API#1301]
Hello  I have a website  www.azz-lottery-syndicate.com
On this page there is a button for people to join the lottery syndicate
When they click on here, I need them to go to check out
at check out i need them to input their details and process the payment
when successful  i need a email to confirm the success and also if it fails
The data i need on the from that comes to me is 
First Name, Second Name, email address, contact telephone number 

So basically simple payment screen 

The account is with vivawallet,  the wallet name is azz-lottery-syndicate  

Please can you call me  on 07738213397 to explain further 

This is a simple process 

Adam

**Comment by highvalueasset** [comment]
[https://azz-lottery-syndicate.com/](url)

**Comment by highvalueasset** [comment]
<!DOCTYPE html>
<html>
    <head>
        <title>Simple Checkout</title>
    </head>
    <body>
      <form id="myform" action="/checkout" method="post">
        <button type="button"
          data-vp-publickey="zCPkxeVyHJaYYcYmIqOqrhKJU6WcdlAwvZH1YgwyR60="
          data-vp-baseurl="https://www.vivapayments.com"
          data-vp-lang="en"
          data-vp-amount="999"
          data-vp-description="Lottery Subscription">
        </button>
      </form>
    

      <script src="https://code.jquery.com/jquery-1.11.2.min.js"></script>
      <script src="https://www.vivapayments.com/web/checkout/js"></script>
    </body>
 
public class TransactionResult
{
    public int ErrorCode { get; set; }
    public decimal Amount { get; set; }
    public string ErrorText { get; set; }
    public Guid TransactionId { get; set; }
}

public async Task<TransactionResult> ChargeAsync(string vivaWalletToken)
{
    var cl = new RestClient("https://www.vivapayments.com/api/") {
        Authenticator = new HttpBasicAuthenticator("fe3a9f82-e0a7-41a8-a560-0f8023ffe1da ", "3a67Qj1zrcOtvwRAB94MerMW2cxAyJ")
    };
    var request = new RestRequest("transactions", Method.POST) {
        RequestFormat = DataFormat.Json
    };

    request.AddParameter("PaymentToken", vivaWalletToken);

    var response = await cl.ExecuteTaskAsync<TransactionResult>(request);

    return response.ResponseStatus == ResponseStatus.Completed &&
        response.StatusCode == System.Net.HttpStatusCode.OK ? response.Data : null;
}
  </html>

what am i doing wrong here 

the first part works fine ,  the second part does not ,  i done the 3rd part ok i think 

please Help

**Comment by highvalueasset** [comment]
<html>
    <head>
        <title>Simple Checkout</title>
    </head>
    <body>
      <form id="myform" action="/checkout" method="post">
        <button type="button"
          data-vp-publickey="zCPkxeVyHJaYYcYmIqOqrhKJU6WcdlAwvZH1YgwyR60="
          data-vp-baseurl="https://www.vivapayments.com"
          data-vp-lang="en"
          data-vp-amount="999"
          data-vp-description="Lottery Subscription">
        </button>
      </form>

      <script src="https://code.jquery.com/jquery-1.11.2.min.js"></script>
      <script src="https://www.vivapayments.com/web/checkout/js"></script>
    </body>
 

  </html>

**Comment by highvalueasset** [comment]
https://azz-lottery-syndicate.com/joining-us

this this page 

I can get to the box appearing and then the card and cv2 and email input then its does nothing 

i tried to put the text above in  but it did not actually work , but showed up as text 

can you help me please 
 
i know all my codes 


Merchant ID | fe3a9f82-e0a7-41a8-a560-0f8023ffe1da
-- | --
API Key | 3a67Qj1zrcOtvwRAB94MerMW2cxAyJ

Public key zCPkxeVyHJaYYcYmIqOqrhKJU6WcdlAwvZH1YgwyR60=

i need the box to be able to go to checkout and get the payment  !

Help Please

**Comment by matthewrgourd** [comment]
Hi @highvalueasset, I'm sorry you're experiencing issues. Unfortunately we can't provide any further technical assistance beyond the [Simple Checkout](https://developer.vivawallet.com/online-checkouts/simple-checkout/) guide within our Developer Portal. Suggest you try Native Checkout v2 if having difficulty implementing Simple Checkout.

**Comment by highvalueasset** [comment]
hello  Matt
I think you missunderstood what I am asking 
the simple checkout , on the website developers i have managed to do that correctly 
the box open up and also i can see my account etc,,, 
on your developer website you say do this stage 2 

stage 2  , 

should i put this on to the same HTML as 1  as when I do that it comes up as text under the button 

should there is something that I need to do to link them in ,  

may be a code 

or am i pasting the item in the wrong please 

that's all I need to do 

if you can direct me please 

![image](https://user-images.githubusercontent.com/59659457/72254670-ae44da00-35fc-11ea-8b83-10b395fbf5fc.png)

so you see what i mean

**Comment by highvalueasset** [comment]
as you can see I have done the simple checkout button on the site 
all i need to do next is to connect it to my account with VIVA

**Comment by highvalueasset** [comment]
So in your development portal   you have stage 1  2 and 3 

i have done 1  and i understand 3 

I just need to know where to cut and paste section 2  and if I need to add any programming text 

please help 

thank you

**Comment by matthewrgourd** [comment]
Hi @highvalueasset unfortunately our checkouts require experience of programming. I'm unable to help you further beyond pointing you in the direction of our documentation. If you want an easier option, I suggest moving your site to an e-commerce platform (like WooCommerce for Wordpress) and installing one of our custom-built [plugins](https://developer.vivawallet.com/e-commerce-plugins/).
